framework/auth/core.py

```diff
@@ -831,6 +905,10 @@ def get_confirmation_token(self, email, force=False):
                 # Old records will not have an expiration key. If it's missing,
                 # assume the token is expired
```

Comment:
It looks like, for old `.verification_key`s, we're assuming them to be expired and (hopefully) resending a confirmation email.

Other than for the password reset flow, when a v1 key is generated solely for communication with CAS, is there any time when a verification key will be generated without an expiration? If not, consider adding a 1 minute expiration to the keys sent to CAS, and deprecating the v1 StringField in favor of a DictionaryField of the same name (requires migration)